railgun/test/integration/controller_test.go

```diff
@@ -0,0 +1,52 @@
+//+build integration_tests
+
+package integration
+
+import (
+	"bytes"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestHealthEndpoint intentionally includes a magic value. localhost:10254 is the default
+// controller health listen. This test should fail, and must be updated, if that default is changed.
+func TestHealthEndpoint(t *testing.T) {
+	_ = proxyReady()
+	assert.Eventually(t, func() bool {
+		resp, err := httpc.Get("http://localhost:10254/healthz")
+		if err != nil {
+			t.Logf("WARNING: error while waiting for http://localhost:10254/healthz: %v", err)
+			return false
+		}
+		defer resp.Body.Close()
+		if resp.StatusCode == http.StatusOK {
+			b := new(bytes.Buffer)
+			b.ReadFrom(resp.Body)
+			return strings.Contains(b.String(), "ok")
```

Comment:
optional:
I'd say that HTTP 200 is sufficient and there's no need to read the body.